### PR TITLE
Update Puzzle.dart

### PR DESCRIPTION
**Pull request by DeepStyles**
hi, I am creating flutter sudoku using your Api, I want this getter to check clock paused or not. its helpful for app if u accept proposal or similar added feature to api.

### CODE REVIEW

File: lib/src/Puzzle.dart [hunk 1]
```diff
@@ -149,6 +149,10 @@ class Puzzle {
   /// I can only make these comments so interesting and no more :l
   void startStopwatch() => _stopwatch.start();
   void stopStopwatch() => _stopwatch.stop();
+  
+  ///to check stopclock is paused or not
+  bool get isStopClockRunning => _stopwatch.isRunning();
```
Comment (AlvinRamoutar) [inline lib/src/Puzzle.dart]:
```suggestion
  bool get isStopwatchRunning => _stopwatch.isRunning();
```

Matching the naming of other `Stopwatch` methods.

**Review by AlvinRamoutar — CHANGES_REQUESTED** [review]

**Comment by AlvinRamoutar** [comment]
Hey @DeepStyles ,
Appreciate the suggestion and contribution!
I've recommended a change, just purely syntactical sugar.

Thanks,
Alvin

**Comment by DeepStyles** [comment]
hey its my first contribution ever thank you...

**Review by DeepStyles — COMMENTED** [review]
sry, previous line throwing error cause isRunning is a getter, so made that change.

**Comment by AlvinRamoutar** [comment]
Hey @DeepStyles , will review sometime this weekend. Thanks!

**Review by AlvinRamoutar — APPROVED** [review]
Approved, @DeepStyles .
Tested this change with the following shortcode:

```
    print(puzzle.isStopwatchRunning);
    puzzle.startStopwatch();
    print(puzzle.isStopwatchRunning);
    sleep(Duration(milliseconds: 1374));
    print(puzzle.getTimeElapsed());
    puzzle.stopStopwatch();
    print(puzzle.isStopwatchRunning);
```

**Pull request merged**